engine/src/test/java/org/camunda/bpm/engine/test/api/runtime/DeleteProcessInstanceUserOperationLogTest.java

```diff
@@ -0,0 +1,155 @@
+import org.camunda.bpm.engine.runtime.ProcessInstance;
+import org.camunda.bpm.engine.test.ProcessEngineRule;
+import org.camunda.bpm.engine.test.RequiredHistoryLevel;
+import org.camunda.bpm.engine.test.util.ProcessEngineTestRule;
+import org.camunda.bpm.engine.test.util.ProvidedProcessEngineRule;
+import org.camunda.bpm.model.bpmn.Bpmn;
+import org.camunda.bpm.model.bpmn.BpmnModelInstance;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+
+@RequiredHistoryLevel(ProcessEngineConfiguration.HISTORY_FULL)
+public class DeleteProcessInstanceUserOperationLogTest {
+
+  private static final String PROPERTY = "property";
+  private static final String NEW_VALUE = "newValue";
+  private static final String PROCESS_INSTANCE_ID = "processInstanceId";
+  private static final String PROCESS_DEFINITION_ID = "processDefinitionId";
+
+  protected ProcessEngineRule engineRule = new ProvidedProcessEngineRule();
+  protected ProcessEngineTestRule testRule = new ProcessEngineTestRule(engineRule);
+
+  @Rule
+  public RuleChain ruleChain = RuleChain.outerRule(engineRule).around(testRule);
+
+  protected ProcessEngineConfigurationImpl processEngineConfiguration;
+  protected RuntimeService runtimeService;
+  protected HistoryService historyService;
+  protected IdentityService identityService;
+  protected RepositoryService repositoryService;
+
+  protected Long defaultLogEntriesPerSyncOperationLimit;
+
+  protected BpmnModelInstance instance;
+  protected Deployment deployment;
+
+  @Before
+  public void setup() {
+    processEngineConfiguration = engineRule.getProcessEngineConfiguration();
+    runtimeService = engineRule.getRuntimeService();
+    historyService = engineRule.getHistoryService();
+    repositoryService = engineRule.getRepositoryService();
+    identityService = engineRule.getIdentityService();
+    defaultLogEntriesPerSyncOperationLimit = processEngineConfiguration.getLogEntriesPerSyncOperationLimit();
+
+    this.instance = Bpmn.createExecutableProcess("process1")
+        .startEvent("start")
+        .userTask("user1")
+        .sequenceFlowId("seq")
+        .userTask("user2")
+        .endEvent("end")
+        .done();
+  }
+
+  @After
+  public void tearDown() {
+    processEngineConfiguration.setLogEntriesPerSyncOperationLimit(defaultLogEntriesPerSyncOperationLimit);
+  }
+
+  @Test
+  public void shouldProduceSingleOperationLog() {
+    // given
+    processEngineConfiguration.setLogEntriesPerSyncOperationLimit(1);
+    testRule.deploy(instance);
+    engineRule.getIdentityService().setAuthenticatedUserId("userId");
+    ProcessInstance instance1 = runtimeService.startProcessInstanceByKey("process1");
+
+    // when
+    runtimeService.deleteProcessInstance(instance1.getId(), null);
+
+    // then
+    UserOperationLogEntry logEntry = historyService.createUserOperationLogQuery().operationType(UserOperationLogEntry.OPERATION_TYPE_DELETE).singleResult();
```

Comment:
🔧 Widescreen monitor ftw. You could add some line breaks 😀 (Below too.)

```suggestion
    UserOperationLogEntry logEntry = historyService.createUserOperationLogQuery()
        .operationType(UserOperationLogEntry.OPERATION_TYPE_DELETE)
        .singleResult();
```